nix/grafana_test.nix

```
@@ -22,8 +76,12 @@
             ADMIN=${cfg.extraConf.security.admin_user}
             PASSWORD=${cfg.extraConf.security.admin_password}
             ROOT_URL="${cfg.protocol}://${cfg.domain}:${builtins.toString cfg.http_port}";
+            # The admin user can authenticate against the running service.
             curl -sSfN -u $ADMIN:$PASSWORD $ROOT_URL/api/org/users -i
             curl -sSfN -u $ADMIN:$PASSWORD $ROOT_URL/api/org/users | grep admin\@localhost
+            # The dashboard provisioner was used to create a dashboard.
+            curl -sSfN -u $ADMIN:$PASSWORD $ROOT_URL/api/dashboards/uid/${dashboardUid} -i
+            curl -sSfN -u $ADMIN:$PASSWORD $ROOT_URL/api/dashboards/uid/${dashboardUid} | grep '"title":${dashboardTitle}'
```

```suggestion
            curl -sSfN -u $ADMIN:$PASSWORD $ROOT_URL/api/dashboards/uid/${dashboardUid} | grep '"title”:"${dashboardTitle}"'
```

`grep` is failing in Linux because `dashboardTitle` is expected to be in `””`.

---

Absolutely!